compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -2279,7 +2279,8 @@ object Parsers {
     /** ConstrApp         ::=  SimpleType {ParArgumentExprs}
      */
     val constrApp = () => {
-      val t = checkWildcard(annotType())
+      // Using Ident(nme.ERROR) to avoid causing cascade errors on non-user-written code
+      val t = checkWildcard(annotType(), fallbackTree = Ident(nme.ERROR))
```

Comment:
In extends caluse you could use `AnyRef` instead of `Any`

Reply:
That didn't work, because I end up with error messages like this, even though `https://github.com/scala/scala/blob/2.13.x/src/library-aux/scala/AnyRef.scala#L15` talks about a `trait AnyRef`:

```scala
scala> class A extends AnyRef with AnyRef
1 |class A extends AnyRef with AnyRef
  |                            ^^^^^^
  |                            class Object is not a trait
```